src/Helper/Address.php

```diff
@@ -58,6 +58,11 @@ public function exists($customerId, array $addressToCheck)
             ->getCollection()
             ->addFieldToFilter('parent_id', $customerId);
 
+        if (empty($addressToCheck['company'])) {
```

Comment:
move this up with the other `addressCheck` manipulation, nicer to have these things in one place if this function grows more.

Reply:
makes sense. done